x-pack/plugins/cases/public/components/cases_context/use_application.tsx

```diff
@@ -18,7 +18,9 @@ export const useApplication = (): UseApplicationReturn => {
   const appId = useObservable(currentAppId$);
   const applications = useObservable(applications$);
 
-  const appTitle = appId ? applications?.get(appId)?.category?.label : undefined;
+  const appTitle = appId
+    ? applications?.get(appId)?.category?.label ?? applications?.get(appId)?.title
```

Comment:
I fixed a small "bug" I found. If the registered application that uses cases does not have a category the `CasesProvider` will not render its children. An app without a category cannot use the `CasesProvider`. I fall back to the title if the category label is not there. This was also needed to the functional tests.